mmocr/models/textrecog/preprocessor/tps_preprocessor.py

```diff
@@ -0,0 +1,168 @@
+# used code from https://github.com/clovaai/deep-text-recognition-benchmark/blob/master/modules/transformation.py
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from mmocr.models.builder import PREPROCESSOR
+from .base_preprocessor import BasePreprocessor
+
+@PREPROCESSOR.register_module()
+class TPSPreprocessor(BasePreprocessor):
+    """ Rectification Network of RARE, namely TPS based STN """
+
+    def __init__(self, F, I_size, I_r_size, I_channel_num=1):
```

Comment:
> F

Typically we do not use a single letter as variable name. Need to make their names more descriptive.